src/Microsoft.DocAsCode.Build.Engine/DocumentBuildContext.cs

```diff
@@ -193,6 +200,92 @@ private List<string> ResolveByExternalReferencePackages(List<string> uidList, Co
             return list;
         }
 
+        private async Task<List<string>> ResolveByXRefServiceAsync(List<string> uidList, ConcurrentDictionary<string, XRefSpec> externalXRefSpec)
+        {
+            if (XRefServiceUrls == null || XRefServiceUrls.Length == 0)
+            {
+                Logger.LogWarning($"You haven't provide an xrefservice item in docfx.json or command options!");
```

Comment:
what should be done or just remove it?

---
In reply to: [126845939](https://github.com/dotnet/docfx/pull/1837#discussion_r126845939) [](ancestors = 126845939)